cmd/util/project.go

```diff
@@ -104,6 +106,10 @@ func (opts *ProjectOpts) GetSignatureKeys() []v1alpha1.SignatureKey {
 	return signatureKeys
 }
 
+func (opts *ProjectOpts) GetSourceNamespaces() []string {
+	return opts.SourceNamespaces
```

Comment:
You mean a nil-check for `opts`? I don't see a reason for this, because imho the caller should make sure to never call any of the receiver's methods on a `nil` object. Or did I miss something here?

Looking at all the other methods for receiver `*ProjectOpts`, none have a nil check. I wonder however if we should make all of the receiver functions to work with a value, not with a pointer, since none of the functions will actually modify the receiver. But probably, that's left for some refactoring out of scope of this PR :)

Reply:
> because imho the caller should make sure to never call any of the receiver's methods on a nil object

Makes sense! Felt like I'd seen some `Get`-style helper functions that did nil checks, but if this is the current pattern, I'm not worried.  :-)